assets/js/util.js

```diff
@@ -9,6 +9,9 @@ var SPACE_KEY_CODE = 32, ENTER_KEY_CODE = 13,
 
 var TEXTAREA_AUTO_RESIZE_MINIMUM_WIDTH = 768;
 
+var amountScrolled = 300; /*:: amount by which user must scroll to view the button */
+var transitionTime = 600; /*:: time taken for the transition to move webpage to top */
```

Comment:
Move these to be part of the var above, use the same variable naming style. No need for comments if the variable names are good.